### DISCUSSION TITLE
Historical Regional Clustering / Исторические регионы

### DISCUSSION THREAD

**Discussion by paul-k-pastvu** [PastVu/pastvu#366] [Ideas / Идеи и предложения]
EN: It was repeatedly proposed to indicate the name of the region on the date of the image. It is clear that there are too many obvious obstacles for its implementation: technical, interface, organizational. Nevertheless, here we briefly state the essence of the proposal:
1. Make a classifier of historical regions, what will be "parallel" to the contemporary one (and optional in relation to it) 
2. For each such region, set the range of action (required) and the borders on the map (optional) 
3. Give the ability to manually relations with such regions. When choosing, only regions matching the image in the date range should be available. 
4. Turn on the filter by historical region in the interface of galleries, subscriptions, and comments (as an additional filter, combined with the main ones) 
5. Grant the moderator rights not only to the contemporary, but also to historical regions. At the same time, for regions without borders, the rights will only apply to those photos that are manually linked to the region

----
RU: Многократно звучало предложение указывать название региона на дату снимка, Понятно, что для ее реализации слишком много очевидных препятствий: технических, интерфейсных, организационных. Тем не менее здесь кратко сформулируем суть предложения:
1. Сделать классификатор исторических регионов, параллельный современному (и дополнительный по отношению к нему)
2. Для каждого такого региона задавать диапазон действия (обязательно) и границы на карте (необязательно)
3. Дать возможность привязывать изображения к таким регионам вручную. При выборе должны быть доступны только регионы, совпадающие со снимком по диапазону дат.
4. Включить фильтр по историческим регионам в интерфейс галерей, подписок и комментариев (как дополнительный фильтр, в сочетании с основными)
5. Давать права модератора не только на основные, но и на исторические регионы. При этом для регионов, у которых не заданы границы, права будут касаться только тех фото, которые привязаны к региону вручную

----
Известные проблемы: те же, что и с фильтром по годам - кластеризация точек на карте.

----
**Dependencies:** #197

**Comment by mkgrgis** [comment]
Исторические данные OHM иногда достигают неплохого качества https://www.openhistoricalmap.org/relation/2693579 Для первичных экспериментов существующего там набора достаточно.
